Tests/PackageLoadingTests/ConventionTests.swift

```diff
@@ -463,16 +463,23 @@ class ConventionTests: XCTestCase {
         let fs = InMemoryFileSystem(emptyFiles:
             "/Sources/Foo/Foo.swift",
             "/Sources/Bar/Bar.swift",
-            "/Sources/Baz/Baz.swift")
+            "/Sources/Baz/Baz.swift",
+            "/Sources/Exec👽/main.swift"
+        )
```

Comment:
Oh forgot about that, it does try to link. Will sep out this change from this PR.